.github/workflows/release.yml

```diff
@@ -14,4 +14,4 @@ jobs:
     with:
       publishToBinaries: true
       mavenCentralSync: true
-      slackChannel: team-lang-cfamily
+      slackChannel: squad-cfamily-notifications
```

Comment:
Quick question: Do you know if we need the `PYTHONUNBUFFERED` environment variable here?

https://github.com/SonarSource/sonar-cpp/blob/369a4bb520181339eecf1303375bf694387a0202/.github/workflows/release.yml#L10-L11